client/src/app/shared/models/event-management/meeting.ts

```diff
@@ -20,12 +24,19 @@ export type AgendaItemCreation = 'always' | 'never' | 'default_yes' | 'default_n
  */
 export type BallotPaperSelection = 'NUMBER_OF_DELEGATES' | 'NUMBER_OF_ALL_PARTICIPANTS' | 'CUSTOM_NUMBER';
 
-export interface Settings {
+const MEETING_NAME = 'meeting';
+export class Settings {
     // Old "general_*" configs
+    @FormClassControl([MeetingAction.CREATE], { required: true })
+    @FormClassControl([MeetingAction.UPDATE])
     name: string;
+    @FormClassControl([MeetingAction.CREATE, MeetingAction.UPDATE])
     description: string;
+    @FormClassControl([MeetingAction.CREATE, MeetingAction.UPDATE])
     location: string;
+    @FormClassControl([MeetingAction.CREATE, MeetingAction.UPDATE])
     start_time: number;
+    @FormClassControl([MeetingAction.CREATE, MeetingAction.UPDATE])
     end_time: number;
```

Comment:
Another approach we have talked about is to use decorators to describe how to create a form-group.